open-idea-station/Acorn
# #155308812 Design notification UI

#### What does this PR do?
It adds a design for notification to the UI

#### Description of Task proposed in this pull request?
- Add a UI design for notifications

#### How should this be manually tested?
- Visit the application
- Login or signup
- Click on the notification icon
- You should see a dropdown of notifications

#### What are the relevant pivotal tracker stories?
- [#155308812](https://www.pivotaltracker.com/story/show/155308812)

#### Any background context you want to add?
N/A

#### What I have learned working on this feature: [If you don't put anything here, you are doing it wrong!]
- I learnt how to combine various designs to get what I want

#### Screenshots: [If you made some visual changes to the application please upload screenshots here, or remove this section]
<img width="1440" alt="screen shot 2018-12-05 at 3 11 38 pm" src="https://user-images.githubusercontent.com/32465536/49518986-3efca280-f8a0-11e8-992f-9fd27fc243cc.png">
